src/app/components/ModalContainer/index.tsx

```
@@ -4,50 +4,50 @@ import { ConvertPokemonUnionSpeciesType } from "@/app/type/pokemon";
+			className="fixed inset-0 z-50 w-full h-full p-0 m-0 bg-transparent"
+		>
+			<div
+				className="fixed inset-0 bg-black/50 backdrop-blur-sm"
+				onClick={handleModelClose}
+			/>
+			<div className="fixed inset-0 overflow-y-auto">
+				<div className="flex items-center justify-center min-h-screen p-4">
+					<div
+						className="relative w-full max-w-2xl border shadow-lg bg-white/80 backdrop-blur-lg rounded-xl border-white/20"
+						onClick={(e) => e.stopPropagation()}
+					>
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)